pkg/storage/storageService.go

```diff
@@ -36,16 +42,18 @@ type commonService struct {
 }
 
 type fcStorage struct {
-	cs commonService
+	cs                commonService
+	connectorInfoPath string
 }
 
 type iscsiStorage struct {
-	cs            commonService
-	iscsiInfoPath string
+	cs                commonService
+	connectorInfoPath string
 }
 
 type sasStorage struct {
```

Comment:
Do these three need to be separate or could we use a common struct? Do we expect to expand into protocol specific info?